lib/Puzzle.js

```diff
@@ -1,47 +1,24 @@
 class Puzzle {
-  constructor(numberOfWords, numberOfLetters, solution) {
-    this.numberOfWords = numberOfWords;
-    this.numberOfLetters = numberOfLetters;
-    this.letterGuess = '';
-    this.solution = solution;
-    this.solved = false
+  constructor(puzzleObject) {
+    // this.numberOfWords = puzzleObject.number_of_words;
+    // this.numberOfLetters = puzzleObject.total_number_of_letters;
+    this.correctAnswer = puzzleObject.correct_answer;
+    // this.category = puzzleObject.category;
+    this.solved = false;
   }
 
-  checkGuess(letterGuess) {
-    this.solution.includes(letterGuess) ? this.letterGuess = letterGuess : 'no match';
-  }
-
-  checkSolution(solutionGuess) {
-    this.solution === solutionGuess ? 'You win the round!' : 'change turn';
-  }
+  // checkGuess(letterGuess) {
+  //   this.solution.includes(letterGuess) ? this.letterGuess = letterGuess : 'no match';
+  //   //should this be in the board? 
```

Comment:
This should be on the puzzle because, if the guess is correct we would need to check the puzzle first, before we update the board, and we want our board to then display the result.

Reply:
after some discussion, we decided that this method probably does not need to be on the Puzzle, just on the Board. Thanks!